cpr/cookies.cpp

```diff
@@ -100,6 +100,10 @@ void Cookies::emplace_back(const Cookie& str) {
     cookies_.emplace_back(str);
 }
 
+bool Cookies::empty() {
```

Comment:
```suggestion
bool Cookies::empty() const {
```

I would suggest making it const.